### PR TITLE
Add TravisCI Setup to README.md

### PR DESCRIPTION
**Pull request by dasilvacontin**

### CODE REVIEW

**Comment by dasilvacontin** [comment]
h/t @hugovk

File: README.md [hunk 1]
```diff
@@ -56,6 +56,19 @@ While ShellCheck is mostly intended for interactive use, it can easily be added
 
 ShellCheck makes canonical use of exit codes, and can output simple JSON, CheckStyle compatible XML, GCC compatible warnings as well as human readable text (with or without ANSI colors). See the [Integration](https://github.com/koalaman/shellcheck/wiki/Integration) wiki page for more documentation.
 
+## TravisCI Setup
```
Comment (hugovk) [inline README.md]: -> "Travis CI"

File: README.md [hunk 2]
```diff
@@ -56,6 +56,19 @@ While ShellCheck is mostly intended for interactive use, it can easily be added
...
+
+If you want to use ShellCheck in TravisCI, setting it up is simple :tada:.
```
Comment (hugovk) [inline README.md]: -> "Travis CI"

File: README.md [hunk 3]
```diff
@@ -56,6 +56,19 @@ While ShellCheck is mostly intended for interactive use, it can easily be added
...
+
+```yml
+language: bash
+addons:
+  apt:
+    sources:
+    - debian-sid    # Grab shellcheck from the Debian repo
```
Comment (hugovk) [inline README.md]: -> "ShellCheck"

**Comment by dasilvacontin** [comment]
@hugovk good catch!

**Comment by koalaman** [comment]
Thanks!

**Pull request merged**